Plugin: 视频、图片解析器

**插件名称：**

视频、图片解析器

**插件功能：**

适用于nonebot2 解析视频、图片链接/小程序插件，tiktok、bilibili、twitter等实时发送！ 

**PyPI 项目名：**

nonebot-plugin-resolver

**插件 import 包名：**

nonebot-plugin-resolver

**插件项目仓库/主页链接：**

https://github.com/zhiyu1998/nonebot-plugin-resolver

**标签：**

[{"label":"bilibili","color":"#f8a5c2"},{"label":"tiktok","color":"#303952"},{"label":"twitter","color":"#1b9cfc"}]

# 📃 商店发布检查结果

> Plugin: 视频、图片解析器

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-resolver 的信息如下：
    name         : nonebot-plugin-resolver                                          
     version      : 1.0.2                                                            
     description  : NoneBot2的解析视频链接/小程序插件，发送 tiktok、bilibili、twitter等可以让QQ发送响应的图片、视频 
    
    dependencies
     - aiohttp &gt;=3.7,&lt;4.0
     - beautifulsoup4 &gt;=4.11,&lt;5.0
     - httpx &gt;=0.23,&lt;0.24
     - lxml &gt;=4.9,&lt;5.0
     - nonebot-adapter-onebot &gt;=2.0.0-beta.1,&lt;3.0.0
     - nonebot2 &gt;=2.0.0-beta.5,&lt;3.0.0
     - tweepy &gt;=4.12,&lt;5.0
插件 nonebot_plugin_resolver 加载出错：
02-16 14:46:45 [SUCCESS] nonebot | NoneBot is initializing...
02-16 14:46:45 [INFO] nonebot | Current Env: prod
02-16 14:46:45 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_resolver&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_resolver&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named &#x27;nonebot_plugin_resolver&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: bilibili-#f8a5c2, tiktok-#303952, twitter-#1b9cfc。</li><li>✅ 项目 <a href="https://github.com/zhiyu1998/nonebot-plugin-resolver">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-resolver/">nonebot-plugin-resolver</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4359,5 +4359,28 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot-plugin-resolver",
+    "project_link": "nonebot-plugin-resolver",
+    "name": "视频、图片解析器",
+    "desc": "适用于nonebot2 解析视频、图片链接/小程序插件，tiktok、bilibili、twitter等实时发送！",
+    "author": "zhiyu1998",
+    "homepage": "https://github.com/zhiyu1998/nonebot-plugin-resolver",
+    "tags": [
+      {
+        "label": "bilibili",
+        "color": "#f8a5c2"
+      },
+      {
+        "label": "tiktok",
+        "color": "#303952"
+      },
+      {
+        "label": "twitter",
+        "color": "#1b9cfc"
+      }
+    ],
+    "is_official": false
   }
 ]

